File: week6/part1/calculator.py
# EXCEPTIONS were made for invalid inputs using TRY and EXCEPT

try:
    x = int(input("x: "))               # cast an int to convert input string to int
except ValueError:
    print("Invalid input. Try again")
    exit()
try:
    y = int(input("y: "))               # without casting an int, 3 + 4 = 34 as string
except ValueError:
    print("Invalid input. Try again")
    exit()

print(x + y)


# Using the cs50 get_int function
"""
from cs50 import get_int

x = get_int("x: ")
y = get_int("y: ")
printf(x + y)
"""

File: week1/calculator.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // int x = get_int("x: ");
    // int y = get_int("y: ");
    // printf ("%i\n", x + y);

    // long x = get_long("x: ");
    // long y = get_long("y: ");
    // printf ("%li\n", x + y);

    float x = get_float("x: ");
    float y = get_float("y: ");
    float z = x / (float) y;
    printf("%.50f\n", z);
}